g/github.com__evanphx__json-patch/github.com__evanphx_json-patch_0.0.0-20190203023257-5858425f7550_rhel_8.4.sh

```diff
@@ -0,0 +1,82 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : Json-patch
+# Version               : v0.0.0-20190203023257-5858425f7550
```

Comment:
does this same script work for top of tree as well? If so, we can remove the version info. from the script name

Reply:
Yes the same script work for top of tree as well,i've removed the version info from the script name.